lib/src/commands/generate/generate_command.dart

```diff
@@ -41,13 +41,16 @@ class GenerateCommand extends Command {
   @override
   Future<void> run() async {
     try {
+      final workingDirectory = argResults!.rest.first;
```

Comment:
Actually in this case, this won't work because there is no path supplied to the generate command.

Please turn your original request into an issue and I'll see if we can get to it next week. We'll be spending some time  working through Stacked issues next week and the week after.